spec/system/passenger_registration_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Passenger self registration', js: true do
+  context 'registering for the first time' do
+    before :each do
+      login_as(build :passenger)
+    end
+    let :submit do
+      click_button 'Submit'
+    end
+    context 'successfully' do
+      it 'creates a new pending passenger' do
+        visit register_passengers_path
+        fill_in 'Address', with: '123 turkey lane'
+        fill_in 'Phone', with: '123'
+        expect{ submit }.to change { Passenger.count }.by 1
+        expect(page).to have_text 'Passenger registration successful'
+        expect(Passenger.last).to be_pending
+      end
+    end
+    context 'unsuccessfully' do
+      it 'renders errors in the flash' do
+        visit register_passengers_path
+        fill_in 'Address', with: '123 turkey lane'
+        expect{ submit }.not_to change { Passenger.count }
```

Comment:
Missing a space before opening bracket
```suggestion
        expect { submit }.not_to change { Passenger.count }
```